packages/editor/src/store/resolvers.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Internal dependencies
+ */
+import {
+	resetAutosave,
+} from './actions';
+import { apiFetch } from './controls';
+
+/**
+ * Request autosave data from the REST API.
+ *
+ * @param {string} postId The Post to retrieve the autosave for.
+ */
+export function* getAutosave( postId ) {
```

Comment:
It seems a bit unusual that the getAutosave resolver requires a postId parameter, but the selector doesn't. Should the autosave state be keyed by postId? That seems unnecessary, but I'm not sure how else to resolve the differences.